src/components/ParserComponent.tsx

```diff
@@ -306,6 +297,21 @@ export default class ParserComponent extends React.Component<ParserInterface,
       }
     }
 
+    /**
+   * creates a new column
```

Comment:
Better docs please. Pre-Post Conditions? Asserts?

Reply:
You're right, on it